safe-room.lic

```diff
@@ -47,6 +57,14 @@ class SafeRoom
     give_and_take(settings.safe_room_id, settings.safe_room_give, settings.safe_room_take)
   end
 
+  def get_health_score(health_hash)
```

Comment:
Or `health_hash.reduce(0) { |sum, (key, array)| sum + key * array.size }`, if I'm reading this right. @rpherbig's version is prettier, but my version only iterates once. Pick your poison.